### ISSUE TITLE
Option to set root folder? [request]

### ISSUE THREAD

**Issue by spiralsam** [salvoventura/save-my-tabs#3]
It looks like the main bookmark root folder from which the popup "Use existing folder" list is taken is from Firefox's default Bookmarks Toolbar. Perhaps have an option to change this?

**Comment by salvoventura** [comment]
@spiralsam That's correct: we code currently looks at the default Bookmarks Toolbar, for simplicity. What would you like the user be able to select instead?

**Comment by emCarCode** [comment]
It would be great to be able to set the root folder as Bookmarks Menu instead of Bookmarks Toolbar since the Menu is where I store the majority of my bookmarks and I only use the Toolbar for very frequently accessed bookmarks.

**Comment by salvoventura** [comment]
Thanks for the additional vote on this. I'll look into this.

**Comment by BeatLink** [comment]
I would love to see this as well!

**Comment by salvoventura** [comment]
Thanks for the additional vote. I admit not having as much time as I used to, but I'll do my best.

**Comment by gitfineon** [comment]
I guess most people would select the 'Bookmarks Menu' as base folder ... I currently use the recently introduced 'Other Bookmarks' folder (`browser.toolbars.bookmarks.showOtherBookmarks`) ... but maybe it is best to let the user select an arbitrary folder :wink:
